deployments/helm-chart/values.yaml

```diff
@@ -271,14 +271,20 @@ controller:
     create: false
 
     ## The spec of the GlobalConfiguration for defining the global configuration parameters of the Ingress Controller.
-    spec: {}
-      # listeners:
-      # - name: dns-udp
-      #   port: 5353
-      #   protocol: UDP
-      # - name: dns-tcp
-      #   port: 5353
-      #   protocol: TCP
+    spec: {
+    # "listeners": [
+    #   {
+    #     "name": "dns-udp",
+    #     "port": 5353,
+    #     "protocol": "UDP"
+    #   },
+    #   {
+    #     "name": "dns-tcp",
+    #     "port": 5353,
+    #     "protocol": "TCP"
+    #   }
+    # ]
+    }
```

Comment:
This looks like a JSON array in a YAML document.
why are we doing it JSON style and not YAML style in a yaml doc.

Reply:
You are right. Since we use `spec: {}` as the default I assumed that we only used JSON notation here. However I seems like the empty curly brace `{}` is only required when leaving the field empty. I've updated the attached issue around this.